### PR TITLE
(Docs) : Added contributing guidelines

### PR DESCRIPTION
**Pull request by Abhijay007**
Added Contributing guidelines

### CODE REVIEW

**Comment by Abhijay007** [comment]
@jhdalek55 review required :)

**Comment by jhdalek55** [comment]
I haven't reviewed this yet, but I'm wondering if we might want to think about the purpose of adding this here. We definitely want people to contribute ideas to the website, but I'm a little unsure about whether we want direct contributions to the text.  This is the face of the project and maybe we want to exercise some control here?  I may be completely wrong about this, so I'll just look at the text for now.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,117 @@ The site is available [here](https://uptane.github.io/).
...
+   ```bash
+   jekyll serve
+   ```
+
+   To preview the site in your web browser navigate to [http://localhost:4000](http://localhost:4000).
+
+
+## Commit changes
+
+After you have a successful testing in local with your changes, you are ready to commit those changes.
```
Comment (jhdalek55) [inline README.md:91]:
```suggestion
After you have successfully tested your changes locally, you are ready to commit those changes.
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
The organization of headers and subheads was not logical. Also, there were a few sentence structure issues.

**Comment by Abhijay007** [comment]
> I haven't reviewed this yet, but I'm wondering if we might want to think about the purpose of adding this here. We definitely want people to contribute ideas to the website, but I'm a little unsure about whether we want direct contributions to the text. This is the face of the project and maybe we want to exercise some control here? I may be completely wrong about this, so I'll just look at the text for now.


@jhdalek55  Thank you for sharing your thoughts on this, I agree with you, I think we might need to create a separate `contributing.md` file for this purpose. Alternatively, we can also add these guidelines to the wiki and provide a link to those documents (`as a link to contributing guidelines`).

However, I would like to point out that these guidelines might change soon as we are still unsure about how we will proceed to improve the site. If we decide to stick with Jekyll, then these guidelines would make sense, but if we need to iterate, we may need to revise them.

For now, I think it would be best to put this on hold, although I will make the changes you suggested.

**Comment by Abhijay007** [comment]
new guidelines are needed as site is updated

**Comment by jhdalek55** [comment]
Sorry. I don't know how to submit a PR now that the MD files are gone.

**Comment by jhdalek55** [comment]
I don't know how to submit the revised text as the MD files are now gone.

**Comment by Abhijay007** [comment]
> Sorry. I don't know how to submit a PR now that the MD files are gone.

you can create a new file contributing.md and can submit the PR with the revised text

**Comment by jhdalek55** [comment]
Done

**Pull request closed without merging**